### DISCUSSION TITLE
Updated web ui but can't test it

### DISCUSSION THREAD

**Discussion by miracle091** [roleoroleo/yi-hack-MStar#352] [Ideas]
Good morning at all, have tried to update the web ui and honestly don't know if it works cause can't test it (don't have the camera source to build the firmware).
List of changes:
- removed most unused code in the index.html
- moved the calls of js scripts a the beginning of the body class
- removed some images (unused), iphone etc
- minified all the images (gif too), they now take up less space
- changed header logo, from png to svg (scale well and look good anyway), name too
- moved favicon from images folder into root, modern browser (IE9+) load it directly without a line in the index
- moved all images from "images" folder to "img", fixed all the calls into the code

[Here the VirusTotal](https://www.virustotal.com/gui/file/2bfdf10800f7673fc8c356a1e68fc5b439c43914fab0bba5edb1b87ebc039832/detection) scan for the zip attached. 

[yi-hack-MStar web-ui updated.zip](https://github.com/roleoroleo/yi-hack-MStar/files/6397865/yi-hack-MStar.web-ui.updated.zip)

P.S. i am trying to implement a dark theme too.

**Comment by roleoroleo** [comment]
I tested it quickly and it seems working.

**Comment by miracle091** [comment]
Finally added the dark mode, but work partially cause the navbar not change and i don't know why...
When i am sure that all things work flawless, i just do a pull request.

List of changes:
- added the dark mode (css + js)
- removed texts on navbar (but retained it into description on the link) and switched to icons (svg from [css.gg](https://github.com/astrit/css.gg))
- minor changes on custom.css

[Here the VirusTotal](https://www.virustotal.com/gui/file/160553df7a743b0f59744280e425a68c8b8836eaea2f2325129098b3ccd450d8/detection) scan for the zip attached. 

[yi-hack-MStar.web-ui.updated.zip](https://github.com/roleoroleo/yi-hack-MStar/files/6415198/yi-hack-MStar.web-ui.updated.zip)
